mri/reconstructors/self_calibrating.py

```diff
@@ -195,18 +204,19 @@ def reconstruct(self, kspace_data, optimization_alg='pogm', x_init=None,
             recompute_smaps = True
         if recompute_smaps:
             # Extract Sensitivity maps and initialize gradient
-            Smaps, _ = get_Smaps(
-                k_space=kspace_data,
-                img_shape=self.fourier_op.shape,
-                samples=self.fourier_op.samples,
-                thresh=self.kspace_portion,
-                min_samples=self.fourier_op.samples.min(axis=0),
-                max_samples=self.fourier_op.samples.max(axis=0),
-                mode=self.smaps_extraction_mode,
-                method=self.smaps_gridding_method,
-                n_cpu=self.n_jobs
-            )
-            self.set_smaps(Smaps)
+            if not check_if_fourier_op_uses_sense(self.fourier_op):
```

Comment:
you could put that on the same line as `recompute_smaps` right ? this would also make it easier to review

Reply:
Yes, I think you are right here, will change